### ISSUE TITLE
Untranslated string when installing a language

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-cms#15503]
### Steps to reproduce the issue
On a NEW install at the final step try to install several additional languages including BULGARIAN

### Expected result
Everything installs

### Actual result
All languages except for bulgarian installs (that might be an error in the language pack) but the bigger "issue" is the untranslated strings in the error

Warning
Joomla was unable to install Bulgarian language. You will be able to install it later using the Joomla! Administrator
JLIB_INSTALLER_ABORT_PACK_INSTALL_ERROR_EXTENSION
JLIB_INSTALLER_ABORT

**Comment by brianteeman** [comment]
### Updated info

When trying to install Bulgarian language pack from the Install Languages option inside Joomla (after successful install) the error is

> Warning
Aborting language installation: Extension must have a title.
Package Install: There was an error installing an extension: site_bg-BG.zip

> Error
Error installing package

So this means that 
1. there is an error in the language pack
2. there is an error in the joomla install when it comes to finding the error strings

**Comment by zero-24** [comment]
hmm looks like the lib language files are not loaded on that page?

**Comment by infograf768** [comment]
any specific lib string has to be added in the installation language file.
we have already a bunch of them there
for example
```
; Framework strings necessary when no lang pack is available
JLIB_DATABASE_ERROR_CONNECT_MYSQL="Could not connect to MySQL."
JLIB_DATABASE_ERROR_DATABASE="A Database error occurred."
JLIB_DATABASE_ERROR_LOAD_DATABASE_DRIVER="Unable to load Database Driver: %s"
JLIB_ENVIRONMENT_SESSION_EXPIRED="Your session has expired, please reload the page."
JLIB_FILESYSTEM_ERROR_COPY_FAILED="Copy failed"
JLIB_FILESYSTEM_ERROR_PATH_IS_NOT_A_FOLDER_FILES="JFolder: :files: Path is not a folder. Path: %s"
JLIB_FORM_FIELD_INVALID="Invalid field:&#160;"
JLIB_FORM_VALIDATE_FIELD_INVALID="Invalid field: %s"
JLIB_FORM_VALIDATE_FIELD_REQUIRED="Field required: %s"
JLIB_INSTALLER_ERROR_FAIL_COPY_FILE="JInstaller: :Install: Failed to copy file %1$s to %2$s."
JLIB_INSTALLER_NOT_ERROR="If the error is related to the installation of TinyMCE language files it has no effect on the installation of the language(s). Some language packs created prior to Joomla! 3.2.0 may try to install separate TinyMCE language files. As these are now included in the core they no longer need to be installed."
JLIB_UTIL_ERROR_CONNECT_DATABASE="JDatabase: :getInstance: Could not connect to database <br />joomla.library: %1$s - %2$s"
```
etc.

**Comment by infograf768** [comment]
as for bg-BG, will look at that pack as I already corrected a few of them yesterday.

**Comment by infograf768** [comment]
For bg-BG, the issue is that , their install.xml name contains only unicode characters.
`<name>Български</name>`
As of 3.7.0, the <name> should contain ascii

**Comment by brianteeman** [comment]
Do you want me to update the language file with the strings or will you

**Comment by brianteeman** [comment]
Closed see #16014
